screenpipe-app-tauri/scripts/pre_build.js

```diff
@@ -46,8 +46,8 @@ const config = {
 		],
 	},
 	macos: {
-		ffmpegName: 'ffmpeg-7.0-macOS-default',
-		ffmpegUrl: 'https://master.dl.sourceforge.net/project/avbuild/macOS/ffmpeg-7.0-macOS-default.tar.xz?viasf=1',
+		ffmpegName: 'ffmpeg-7.1',
+		ffmpegUrl: 'https://evermeet.cx/ffmpeg/ffmpeg-7.1.7z',
```

Comment:
@m13v why did you change? i guess something with merge/rebase

i recommend rebasing in general